waiter/src/waiter/process_request.clj

```diff
@@ -168,9 +168,7 @@
 (defn request->endpoint
```

Comment:
Is this function needed anymore. Can we inline the `uri` at its use?

Reply:
inlined